src/components/structures/MessagePanel.tsx

```diff
@@ -467,6 +467,17 @@ export default class MessagePanel extends React.Component<IProps, IState> {
 
     // TODO: Implement granular (per-room) hide options
     public shouldShowEvent(mxEv: MatrixEvent, forceHideEvents = false): boolean {
```

Comment:
Would be good to rename this, because it sounds like it's the opposite of shouldHideEvent, but it's not.

Reply:
Agree with you, but will probably leave that one out of this PR as I don't want to disturb that area of the codebase more than I currently have